client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.html

```diff
@@ -109,15 +112,15 @@
                     <span class="oui-status oui-status_warning"
                         data-ng-if="$row.isPublic() && $ctrl.hasObsoleteBandwithOption()">
                         <span class="oui-icon oui-icon-warning_circle oui-color-warning-medium font-inherit"></span>
-                        <span data-ng-bind="'server_bandwidth_type_' + $ctrl.specifications.bandwidth.type | translate"></span>
+                        <span data-ng-bind="'dedicated_server_bandwidth_type_' + $ctrl.specifications.bandwidth.type | translate"></span>
```

Comment:
We did the same on each of our PR :) => https://github.com/ovh-ux/ovh-manager-dedicated/pull/1433/files#diff-94b10a4c26cb0578ba6f1f52644efba5R112